src/main/java/organice/logic/commands/DoneCommand.java

```diff
@@ -0,0 +1,107 @@
+package organice.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static organice.logic.commands.MatchCommand.match;
+
+import organice.model.Model;
+import organice.model.person.Donor;
+import organice.model.person.Nric;
+import organice.model.person.Patient;
+import organice.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Set the status of a donor and patient to Done and remove the pair from the model
+ */
+public class DoneCommand extends Command {
+    public static final String COMMAND_WORD = "done";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finish the processing of patients and donors"
+            + "Parameters: ic/PATIENT NRIC ic/DONOR NRIC \n"
+            + "Example: " + COMMAND_WORD + " ic/s4512345A ic/s7711123C";
+
+    public static final String MESSAGE_SUCCESS = "Done processing the patient and donor";
+    public static final String MESSAGE_NOT_PROCESSED = "Donor or patient NRIC must be valid";
+
+    private String firstNricString;
+    private String secondNricString;
+
+    private Nric firstNric;
+    private Nric secondNric;
```

Comment:
i dont get why there are 3 diff vars abt nric `firstNric` and `patientNric` any diff? can just use `.toString()`?

Reply:
Because the thing accepts 2 nric. We need to identify whixh nric is patient and which is donor and convert then frm string to nric